test/jdk/java/net/HttpURLConnection/SetAuthenticator/HTTPTestServer.java

```diff
@@ -978,6 +978,7 @@ static class HttpsProxyTunnel extends HTTPTestServer
             implements Runnable {
 
         final ServerSocket ss;
+        private volatile boolean stop;
```

Comment:
Please add empty line. It might be a clean backport then.

Reply:
fixed